### DISCUSSION TITLE
Check the render method of `Controller`.Did you mean to use React.forwardRef()?

### DISCUSSION THREAD

**Discussion by BekaArabidze** [react-hook-form/react-hook-form#4198] [Q&A]
Basically, my problem is when I use Controller in react-hook-form. I am using Geist-UI(Auto-complete component) and could not solve the problem. There are lots of discussions already but nothing helpful. How can I fix this?
![Screenshot_1](https://user-images.githubusercontent.com/36812744/107923925-9a03a700-6f8b-11eb-97cc-92745857c9fd.png)
![Screenshot_2](https://user-images.githubusercontent.com/36812744/107924019-bdc6ed00-6f8b-11eb-9c75-0e72ca92bc74.png)

**Comment by bluebill1049** [comment]
you don't need to use register when using `Controller`, it does that for you.

**Reply by BekaArabidze** [reply]
that's true, but the problem still remains

**Reply by gepd** [reply]
This may be because the component does not exposes the ref property?

**Reply by davepoon** [reply]
Probably you can wire up the `ref` to the AutoComplete component, this is the old comment https://github.com/react-hook-form/react-hook-form/issues/3411#issuecomment-724992358 posted by bluebill1049, perhaps this way could solve your issue.

**Reply by uttampun44** [reply]
![Screenshot from 2024-07-23 20-44-07](https://github.com/user-attachments/assets/c4e9fad7-e790-4feb-8454-ab741c31a533)
![Screenshot from 2024-07-23 20-43-55](https://github.com/user-attachments/assets/1b4d1b0c-43ea-4092-9a0a-b6523fab0a1d)
before register Remove the register; it will work. I faced the same issue. after removing register 

![Screenshot from 2024-07-23 20-44-28](https://github.com/user-attachments/assets/5772d50d-6939-4dca-918e-3360e021dadc)
![Screenshot from 2024-07-23 20-44-19](https://github.com/user-attachments/assets/2302b870-a542-4ab8-a623-376af7892f59)

**Comment by Mendoza920** [comment]
Hello apparently many have been using my account.
